sipvicious/svcrack.py

```diff
@@ -546,8 +577,11 @@ def main():
         sipvicious.start()
         if exportpath is not None:
             open(os.path.join(exportpath, 'closed'), 'w').close()
+
     except KeyboardInterrupt:
         logging.warning('caught your control^c - quiting')
+        __exitcode__ = resolveexitcode(30, __exitcode__)
```

Comment:
keyboard interrupt is not a network error.

No exit code should be set on keyboard interrupt; any previous exit codes set should follow through to the final `exit()` whether or not this exception is triggered.

Reply:
taken care in 3d909a7.